File: 151001/Kononova/rest_lab/src/main/java/by/bsuir/entities/Message.java
package by.bsuir.entities;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Message {

    private Long id;
    private Long issueId;
    private String content;
}

File: 151004/Afanasenko/appLab1/src/main/kotlin/com/github/kornet_by/dc/lab1/bean/Message.kt
package com.github.kornet_by.dc.lab1.bean

import com.github.kornet_by.dc.lab1.dto.response.MessageResponseTo
import kotlinx.serialization.Serializable

@Serializable
data class Message(
	val id: Long, val issueId: Long, val content: String
) {
	fun toResponse(): MessageResponseTo = MessageResponseTo(id, issueId, content)
}